cli/dcoscli/data/help/package.txt

```diff
@@ -3,73 +3,82 @@ Install and manage DCOS packages
 Usage:
     dcos package --config-schema
     dcos package --info
+    dcos package create [--output-directory=<output-directory>]
```

Comment:
Yep. We will implement `export` and `import` this change only implements `create`